### DISCUSSION TITLE
Can I build osgEarth with Visual studio 2015 update3

### DISCUSSION THREAD

**Discussion by chuyanjuncn** [pelicanmapping/osgearth#2625] [Support Q&A]
Can I build osgEarth with Visual studio 2015 update3.

**Comment by gwaldron** [comment]
I don't see why not. Did you try it?

**Comment by chuyanjuncn** [comment]
yes,  I try it, but there are error  when building.

**Comment by chuyanjuncn** [comment]
F:\osgearth-osgearth-3.6\src\osgEarth\ImageLayer.cpp(882): error C2440: “initialize ”: can not convert “initializer list”to“jobs::context”
F:\build\vcpkg_installed\x64-windows\include\absl/base/policy_checks.h(50): fatal error C1189: #error:  "This package requires Visual Studio 2019 (MSVC++ 16.0) or higher." 
F:\build\vcpkg_installed\x64-windows\include\absl/base/policy_checks.h(50): fatal error C1189: #error:  "This package requires Visual Studio 2019 (MSVC++ 16.0) or higher."
F:\osgearth-osgearth-3.6\src\osgEarth\TDTiles.cpp(558): error C2440: “initialize”: can not convert “initializer list ”to“ jobs::context”
F:\osgearth-osgearth-3.6\src\osgEarthDrivers\cache_filesystem\FileSystemCache.cpp(677): error C2440: “initialize ”: can not “initializer list” to “jobs::context”

**Comment by gwaldron** [comment]
I presume you are using vcpkg?
Looks to me like one of the vcpkg dependencies ([abseil](https://github.com/abseil/abseil-cpp)?) is requiring a minimum version of Visual Studio. Not much we can do about that I'm afraid; you can upgrade, or perhaps try to build the dependencies yourself against an older version. Good luck.
